### ISSUE TITLE
In-page navigation

### ISSUE THREAD

**Issue by rebeccacremona** [harvard-lil/capstone#480]
As targets for in-page navigation, cap is using a combination of named anchors (`<a name="thing"></a>`) and anchor tags lacking `href` attributes, but with `id`s (`<a id="thing"></a>`)

`name` was valid on anchors in previous versions of html, but it's actually not anymore in html5. Instead, you are supposed to point in-page navigation at the id of the element you want to jump to, usually a heading. `<a href="thing">Thing Section</a>......<h2 id="thing">Thing Section</h2>`

I bet that's why some in-page links here point to anchors with ids, like `<a id="thing"></a>`. But actually, anchor tags without `href` attributes are also not allowed in html5 LOL!

With the hardware/software I have at hand, this doesn't seem to be causing a problem: in-page links of both kinds are working on my mac w/ all my browsers, for keyboard and mouse and keyboard + VoiceOver. I could believe that this would be broken for other hardware/software combos: JAWS + IE, for instance.

Probably not a high priority, but it's worth it at some point changing all in-page navigation to point to unique ids of heading elements.

**Comment by ChefAndy** [comment]
Resolved.
